src/legacy/core_plugins/data/public/search/aggs/buckets/geo_hash.ts

```diff
@@ -18,69 +18,22 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { geohashColumns } from 'ui/vis/map/decode_geo_hash';
-import chrome from 'ui/chrome';
 import { BucketAggType, IBucketAggConfig } from './_bucket_agg_type';
 import { KBN_FIELD_TYPES } from '../../../../../../../plugins/data/public';
-
-import { geoContains, scaleBounds, GeoBoundingBox } from './lib/geo_utils';
 import { BUCKET_TYPES } from './bucket_agg_types';
-import { AggGroupNames } from '../agg_groups';
 
-const config = chrome.getUiSettingsClient();
+const defaultBoundingBox = {
+  top_left: { lat: 1, lon: 1 },
+  bottom_right: { lat: 0, lon: 0 },
+};
 
 const defaultPrecision = 2;
-const maxPrecision = parseInt(config.get('visualization:tileMap:maxPrecision'), 10) || 12;
```

Comment:
all this logic was moved inside tile_map_vis (it needs to set correct precision based on zoom level)